src/filters.js

```diff
@@ -117,7 +117,7 @@ var filters = {
 
     escape: function(str) {
         if(typeof str === 'string' ||
-           str instanceof r.SafeString) {
+           str instanceof r.SafeString === false) {
```

Comment:
This definitely doesn't look right; it would cause this filter to attempt to escape any object which is not a `SafeString`. I think instead the entire conditional can be simplified to just `if(typeof str === 'string')` removing the `||` entirely.